DOC linkfield accesibility improvements (#479)
silverstripe/developer-docs: en/08_Changelogs/5.3.0.md

This is particularly helpful if you need to update columns in one table to match values from another table.

### Other new features

> - `silverstripe/linkfield` now has improved accesibilty support for screen readers and keyboard navigation. Also, focus states have been made consistent when using a keyboard or a mouse.

Reviewer: ```suggestion
- `silverstripe/linkfield` now has improved accessibility support for screen readers and keyboard navigation. Focus states have also been made consistent between keyboard and mouse interaction.
```
The latter part of that is really just a subjective rewording so feel free to take or leave it - the main thing here is the typo for "accessibility"

Author: Updated
